packages/number-field/src/NumberField.ts

```diff
@@ -340,6 +340,13 @@ export class NumberField extends TextfieldBase {
                     new Event('change', { bubbles: true, composed: true })
                 );
                 break;
+            case 'Enter':
+                event.preventDefault();
```

Comment:
What is the default that you're preventing here?

We haven't gotten to Form Association for these Custom Elements, but in the future, were this associated to a form, there would be some expectation that `Enter` would submit the form by default.

Reply:
good point! I have removed the `event.preventDefault()` and I've updated the few tests with the current behaviour expectations